src/mod.ts

```diff
@@ -56,6 +56,20 @@ export type ChatMembersOptions = {
    * another chat where the bot is present.
    */
   getKey: (chatId: string | number, userId: number) => string;
+  /**
+   * Configuration options for caching chat members
+   * @default { enabled: false, ttl: 60 }
+   */
+  caching?: {
+    /**
+     * Whether or not to enable caching of chat members
+     */
+    enabled: boolean;
+    /**
+     * Time in seconds to keep chat members in cache
+     */
+    secondsToLive?: number;
```

Comment:
Needs to be removed now

Reply:
Hmm... maybe renamed. The idea is that the chat member gets refetched after this period of time, to prevent outdated information. It's not so much about purging.

Reply:
For aggressive mode, we don't want to fetch before this time. And we do want to fetch after this time.

Reply:
> Even for regular usage, chat_member updates won't inform the bot of, say, name changes.

oh no

Reply:
Right, then I guess there's no way around having this option

Reply:
About applying it to database records as well, that'd be breaking, because I'd need to change the structure of what's stored in it.

Reply:
It's an unrelated change anyway. Let's keep this one for caching only. We can do breaking changes in a different PQ once we're ready for a new major version.

Reply:
Hm... But if we have the cache outdated, we will refetch from the db, which will also be outdated since it doesn't have a ttl.
So I guess you're right, it makes no sense to have this for now.
I can work on a different refetch strategy on a different PQ.
Let's just merge this as an MVP now and worry about refetching later

Reply:
Should we turn caching on by default when aggressive mode is enabled?

Reply:
Yes